myuw/templates/handlebars/card/registration/reg_resources.html

```diff
@@ -39,7 +39,7 @@ <h4 class="sr-only">Registration resources</h4>
             {{/if}}
             <li>
                 <a target="_blank"
-                   href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/">MyPlan</a>
+                   href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/">MyPlan</a> <span>(Non-PCE courses only)</span>
```

Comment:
Make the '(Non-PCE courses only)' inside </a> to be consistent with other link styles in MyUW.